packages/server/lib/modes/run.ts

```diff
@@ -657,7 +657,7 @@ async function waitForTestsToFinishRunning (options: { project: Project, screens
 
   if (!shouldUploadVideo) {
     debug(`Spec run had no failures and config.videoUploadOnPasses=false. Skip compressing video. Video path: ${videoName}`)
-    results.video = null
```

Comment:
why did we remove `results.video = null`?

Reply:
because video is still be captured, it just isn't uploaded to the cloud or compressed, meaning that we still want to print video: true in the terminal but want to return null when the function ends, which is handled [here](https://github.com/cypress-io/cypress/pull/27008/files#diff-23312a21d720a74c51c81d569ca48200ab80885b2f4415df78e39cea5685788fR727)

Reply:
ultimately it doesn't matter too much, since this gets ripped out in #27010